docs/mindsdb-docs/docs/connect/dbt.md

```diff
@@ -44,7 +44,7 @@ To integrate your predictions into your DBT workflow, use the dbt-mindsdb adapte
           type: mindsdb
           database: 'mindsdb'
           host: 'cloud.mindsdb.com'
-          port: 47335
+          port: 3306
```

Comment:
While testing the connection, I came across these guys (this one, and the one in `mysql-client`) that, at least for me, are not working. Happy to drop them from the PR if they work for everyone else.

Reply:
Thanks, this is correct `47335` is for local installation